backend/src/email/middlewares/email.middleware.ts

```diff
@@ -161,8 +161,10 @@ const existsFromAddress = async (
 ): Promise<Response | void> => {
   const { from } = req.body
   const defaultEmail = config.get('mailFrom')
-  if (from === defaultEmail) return next()
-  const { fromName, from: fromAddress } = res.locals
+  const { fromName, from: fromAddress, templateFrom } = res.locals
+  // Skip check if either the provided from address or the template's from address is the same
+  // as the configured default mail from.
+  if (from === defaultEmail || templateFrom === defaultEmail) return next()
```

Comment:
Discovered a bug while re-testing. Campaigns using the default from address were not able to be sent. The checks were not skipped as `from` will be `null` for `POST /credentials` and `POST /send`. 

`templateFrom` has to be added because `defaultFrom` takes the form of `Postman.gov.sg <donotreply@mail.postman.gov.sg>`. 

@jeantanzj do you think this check can be better handled?